### PR TITLE
US19633/Adds content menu according to rebrand

### PR DESCRIPTION
**Pull request by kellgauthier**
## Task
Reliant on crds-styles [pull request](https://github.com/crdschurch/crds-styles/pull/456). 
Adds explore our content menu per rebrand. 

[Rally](https://rally1.rallydev.com/#/66096747656ud/detail/userstory/395842984940?fdp=true)

## Notes 
Per AG - Messages link has been removed. 
Screen Shot 2020-06-02 at 12 31 46 PM

### CODE REVIEW

File: _includes/home/_content-menu.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div class="container">
+    <div class="col-md">
+        <section class="content-menu">
+            <p class="menu-title soft-left font-family-serif text-uppercase">Explore All of Our Content</p>
+                <ul style="list-style: none; padding-left: 0px; padding-top: 4px;">
```
Comment (silasbaur) [inline _includes/home/_content-menu.html:5]: I don't think these inline styles are necessary. you could use these classes `list-unstyled hard-left soft-quarter-top`

File: _includes/home/_content-menu.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div class="container">
+    <div class="col-md">
```
Comment (tcmacdonald) [inline _includes/home/_content-menu.html:2]: Where is `col-md` defined? That looks like a BS4 selector but this site uses BS3.

File: _includes/home/_content-menu.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        <section class="content-menu">
+            <p class="menu-title soft-left font-family-serif text-uppercase">Explore All of Our Content</p>
+                <ul class="list-unstyled hard-left soft-quarter-top"> 
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="/media/series">Messages</a></li>
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="/media/series/">Series</a></li>
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="https://www.crdsmusic.com/">Music</a></li>
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="/media/podcasts/">Podcasts</a></li>
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="/media/articles/">Articles</a></li>
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="/media/videos/">Videos</a></li>
+                </ul>
```
Comment (tcmacdonald) [inline _includes/home/_content-menu.html:12]: Please fix your indentions– you should be using soft-tabs w/ spaces (two spaces per tab).

**Comment by tcmacdonald** [comment]
Closed in lieu of crdschurch/crds-net#1550

**Pull request closed without merging**